awx/main/credential_plugins/hashivault.py

```diff
@@ -224,6 +245,32 @@ def ssh_backend(**kwargs):
     return resp.json()['data']['signed_key']
 
 
+def token_backend(**kwargs):
+    token = handle_auth(**kwargs)
+    url = urljoin(kwargs['url'], 'v1')
+    cacert = kwargs.get('cacert', None)
+
+    request_kwargs = {'timeout': 30}
+    if cacert:
+        request_kwargs['verify'] = create_temporary_fifo(cacert.encode())
```

Comment:
FYI, I'm about to merge a PR somewhat related to this that is likely to conflict - we're getting out of the habit of managing the certificate data in this way:

https://github.com/ansible/awx/pull/7175